storage/storage.py

```diff
@@ -8,7 +8,7 @@
 
 from share import ExpandEventListFromField, ProtocolMultiline
 
-CHUNK_SIZE: int = 1024
+CHUNK_SIZE: int = 1024**2
```

Comment:
this is quite a significant change - a comment could be useful as to the significance of this number

Reply:
yes indeed!